content/en-US/api/fontVariantCaps.md

```diff
@@ -0,0 +1,101 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontVariantCaps'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontVariantCaps
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontVariantCaps
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps).**
+
+## Description
+
+The `fontVariantCaps` property of `CanvasRenderingContext2D` allows developers
+to control the use of alternate glyphs for capital letters. If multiple sizes 
+of capital letter glyphs are available for the chosen font, this 
+`CanvasRenderingContext2D` property will choose the one with appropriate size. 
+Otherwise it synthesizes small-caps by using uppercase glyphs. The default 
+value is `normal`.
+
+
+## Syntax
+
+`CanvasRenderingContext2D.fontVariantCaps = "normal" || "small-caps" || "all-small-caps" || "petite-caps" || "all-petite-caps" || "unicase" || "titling-caps"`
+`var fontVariantCaps = CanvasRenderingContext2D.textRendering`
+
+### Values
+
+A `string` representing current `fontVariantCaps` value. The possible values are:
```

Comment:
```suggestion
A `string` representing current the `fontVariantCaps` value. The possible values are:
```

Reply:
I messed up the suggestion and put 'the' in the wrong place. Here's what I should have done.

By the way, do you not see a "Commit suggestion" button?